templates/pages/links.html

```diff
@@ -4,10 +4,16 @@
 
 {% block content%}
 
-	<div>
-		<p>We gather useful links to tutorials, good practices, etc. </p>
-
-	</div>
-
+    <div>
+		<p><h3>Useful links to design ideas, tutorials, good practices and more: </h3></p>
```

Comment:
Again, the `<div>` and `<p>` tags are not necessary, just keep (and style) `<h3>`.